packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2500,8 +2505,9 @@ export class StripeHelper extends StripeHelperBase {
       invoiceNumber,
       invoiceTotalInCents,
       invoiceTotalCurrency,
-      invoiceSubtotalInCents,
+      invoiceSubtotalInCents: showSubtotal ? invoiceSubtotalInCents : null,
       invoiceDiscountAmountInCents,
+      invoiceTaxAmountInCents: invoiceTaxAmountInCents ?? null,
```

Comment:
@pdehaan Yes, but I was thinking we should also catch the edge case in which the left side is `0` (I'm not sure if that could ever happen), but I wanted to avoid creating a redundant line of "Taxes and fees: $0.00" (or whatever the currency). However, I think I misread the doc and the logical OR operator (||) is actually what should be used to cover that case as well instead the nullish coalescing operator, correct?

@StaberindeZA, sorry for making this complicated!

Reply:
No problems. I'm racking my brain trying to remember why I had my initial logic in there. Now that we've added a `showTaxAmount` boolean, I think it's safe to just return the value as is from Stripe, so I'll make that change. Thanks both for checking!